packages/adaptive/src/client.ts

```diff
@@ -0,0 +1,53 @@
+import type { BucketClient } from '@bucketco/browser-sdk';
+import Cookies from 'js-cookie';
+
+const COOKIE_NAME = 'gitbook-visitor-public-bucket';
```

Comment:
I don't understand why the cookie name contains `bucket`. Is the idea to read ALL cookies that start with the prefix `gitbook-visitor-public-`?

Reply:
Yes. GBO will read all cookies with that prefix.